src/pk-main.c

```diff
@@ -182,7 +182,16 @@ main (int argc, char *argv[])
 	syslog (LOG_DAEMON | LOG_DEBUG, "daemon start");
 
 	/* after how long do we timeout? */
-	exit_idle_time = g_key_file_get_integer (conf, "Daemon", "ShutdownTimeout", NULL);
+	exit_idle_time = g_key_file_get_integer (conf, "Daemon", "ShutdownTimeout", &error);
+	/* THIS COMMENT IS A TSUNAMI STONE
+         * The automatic shutdown timeout is necessary to avoid very high
```

Comment:
Technically this is band-aid... Why not
```suggestion
	 * The automatic shutdown is used to prevent memory leaks in some backends to get out of hand. If you want to remove the timeout, please ensure you are not regressing Red Hat bugzilla #1354074.
```

Reply:
I removed the implication that the timeout was only relevant to dnf, but kept the slightly salty "(again)" and the suggestion to go digging past the first git blame.  I want anyone who comes along, thinking to remove the timeout, to understand that the exact thing that they want to do has caused problems more than once.